### PR TITLE
add authors to article front matter

### LINKED ISSUE #216: add blog post authors to article front matter and align authors.yml

### PR DESCRIPTION
**Pull request by qrkourier**

### CODE REVIEW

File: docusaurus/blog/authors.yml [hunk 1]
```diff
@@ -1,17 +1,23 @@
...
+  title: OpenZiti Maintainer
+  url: https://hashnode.com/@andrewpmartinez
+  image_url: /img/profile-andrew-martinez.webp
 
-yangshun:
-  name: Yangshun Tay
-  title: Front End Engineer @ Facebook
-  url: https://github.com/yangshun
-  image_url: https://github.com/yangshun.png
+paul_lorenz:
```
Comment (dovholuknf) [inline docusaurus/blog/authors.yml]: would there be value in using the github handle? feels that way but maybe not?

File: docusaurus/blog/authors.yml [hunk 2]
```diff
@@ -1,17 +1,23 @@
-endi:
-  name: Endilie Yacop Sucipto
-  title: Maintainer of Docusaurus
-  url: https://github.com/endiliey
-  image_url: https://github.com/endiliey.png
+andrew_martinez:
+  name: Andrew Martinez
+  title: OpenZiti Maintainer
+  url: https://hashnode.com/@andrewpmartinez
```
Comment (dovholuknf) [inline docusaurus/blog/authors.yml]: probably the github profile instead

File: docusaurus/blog/authors.yml [hunk 3]
```diff
@@ -1,17 +1,23 @@
...
+  image_url: /img/profile-andrew-martinez.webp
```
Comment (dovholuknf) [inline docusaurus/blog/authors.yml]: i think if you use the github profile these aren't needed

File: docusaurus/blog/authors.yml (hunk 1 above)
Comment (qrkourier) [reply docusaurus/blog/authors.yml]: Done

**Review by dovholuknf — APPROVED** [review]

**Pull request merged**